app/controllers/api/articles_controller.rb

```diff
@@ -7,11 +7,28 @@ def index
   end
 
   def show
-    begin
-      article = Article.find(params[:id])
-      render json: article, serializer: Article::ShowSerializer
-    rescue
-      render json: { message: "Article with id #{params[:id]} could not be found." }, status: :not_found
+    article = Article.find(params[:id])
+    render json: article, serializer: Article::ShowSerializer
+  rescue StandardError
+    render json: { message: "Article with id #{params[:id]} could not be found." }, status: :not_found
+  end
+
+  def create
+    article = Article.create(article_params)
+    if article.persisted?
+      render json: { id: article.id, message: 'Article successfully created!' }
+    elsif article_params['title'].nil?
```

Comment:
You dont need these specific error handlers since u have validations for those attributes on the model. 

You are over complicating it

Reply:
The purpose is to let the front end know what went wrong. Do they have that information in any other way?

Reply:
If you have good suggestion on other solution, please share, we spent some time on this, and over complicated it even more than this before reverting.

Reply:
put a binding pry in the else block and take a look at the errors on the article variable

Reply:
we used rails errors to create error json messages, updated.